runtime/compiler/control/CompilationThread.cpp

```diff
@@ -8787,13 +8788,20 @@ TR::CompilationInfoPerThreadBase::compile(
                {
                TR_ASSERT(_compiler.getHotnessName(_compiler.getMethodHotness()), "expected to have a hotness string");
                if (_compiler.getOutFile() != NULL && _compiler.getOption(TR_TraceAll))
-                  traceMsg(&_compiler, "<compile\n"
-                          "\tmethod=\"%s\"\n"
-                          "\thotness=\"%s\"\n"
-                          "\tisProfilingCompile=%d>\n",
-                          _compiler.signature(),
-                          _compiler.getHotnessName(_compiler.getMethodHotness()),
-                          _compiler.isProfilingCompilation());
+                  {
+                  traceMsg(&_compiler, "<compile\n");
+                  traceMsg(&_compiler, "\tmethod=\"%s\"\n", _compiler.signature());
+                  traceMsg(&_compiler, "\thotness=\"%s\"\n", _compiler.getHotnessName(_compiler.getMethodHotness()));
+                  traceMsg(&_compiler, "\tisProfilingCompile=%d", _compiler.isProfilingCompilation());
+#if defined(JITSERVER_SUPPORT)
+                  if (_compiler.isOutOfProcessCompilation() && TR::compInfoPT->getClientData()) // using jitserver && client JVM
+                     {
+                     traceMsg(&_compiler, "\n");
+                     traceMsg(&_compiler, "\tclientID=%" PRIu64 "", TR::compInfoPT->getClientData()->getClientUID());
```

Comment:
This line has the same issue: extra `""` which can be removed

Reply:
Thanks, I also edited this.